packages/camera/camera_windows/lib/camera_windows.dart

```diff
@@ -0,0 +1,437 @@
+        );
+      }).toList();
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<int> createCamera(
+    CameraDescription cameraDescription,
+    ResolutionPreset? resolutionPreset, {
+    bool enableAudio = false,
+  }) async {
+    try {
+      final Map<String, dynamic>? reply = await _pluginChannel
+          .invokeMapMethod<String, dynamic>('create', <String, dynamic>{
+        'cameraName': cameraDescription.name,
+        'resolutionPreset': resolutionPreset != null
+            ? _serializeResolutionPreset(resolutionPreset)
+            : null,
+        'enableAudio': enableAudio,
+      });
+
+      if (reply == null) {
+        throw CameraException('System', 'Cannot create camera');
+      }
+
+      return reply['cameraId']! as int;
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Future<void> initializeCamera(
+    int cameraId, {
+    ImageFormatGroup imageFormatGroup = ImageFormatGroup.unknown,
+  }) async {
+    final int requestedCameraId = cameraId;
+
+    /// Creates channel for camera events.
+    _cameraChannels.putIfAbsent(requestedCameraId, () {
+      final MethodChannel channel =
+          MethodChannel('flutter.io/cameraPlugin/camera$requestedCameraId');
+      channel.setMethodCallHandler(
+        (MethodCall call) => handleCameraMethodCall(call, requestedCameraId),
+      );
+      return channel;
+    });
+
+    final Map<String, double>? reply;
+    try {
+      reply = await _pluginChannel.invokeMapMethod<String, double>(
+        'initialize',
+        <String, dynamic>{
+          'cameraId': requestedCameraId,
+        },
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+
+    cameraEventStreamController.add(
+      CameraInitializedEvent(
+        requestedCameraId,
+        reply!['previewWidth']!,
+        reply['previewHeight']!,
+        ExposureMode.auto,
+        false,
+        FocusMode.auto,
+        false,
+      ),
+    );
+  }
+
+  @override
+  Future<void> dispose(int cameraId) async {
+    if (_cameraChannels.containsKey(cameraId)) {
+      final MethodChannel? cameraChannel = _cameraChannels[cameraId];
+      cameraChannel?.setMethodCallHandler(null);
+      _cameraChannels.remove(cameraId);
+    }
+
+    try {
+      await _pluginChannel.invokeMethod<void>(
+        'dispose',
+        <String, dynamic>{'cameraId': cameraId},
+      );
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  @override
+  Stream<CameraInitializedEvent> onCameraInitialized(int cameraId) {
+    return _cameraEvents(cameraId).whereType<CameraInitializedEvent>();
+  }
+
+  @override
+  Stream<CameraResolutionChangedEvent> onCameraResolutionChanged(int cameraId) {
+    /// Windows camera plugin does not support resolution changed events.
```

Comment:
Document why. e.g. Windows APIs don't support resolution changing vs we haven't yet implemented this (in which case please file a bug and rephrase as a TODO).

Reply:
@cbracken just added TODOs for each of these for now.
Is is ok to file a bugs before this source is landed, as I see it a bit problematic to have loose issues without source to point to.

Reply:
It's fine to file issues without the source landed; that's how almost all TODO issue links work. You can reference the PR instead (e.g., "https://github.com/flutter/plugins/pull/4641 adds support for Windows but <foo> isn't yet implemented. [...]")